ci-operator/config/openshift/service-serving-cert-signer/master.json

```diff
@@ -0,0 +1,50 @@
+{
+  "tag_specification": {
+    "cluster": "https://api.ci.openshift.org",
+    "namespace": "openshift",
+    "name": "origin-v3.11",
+    "tag": "",
+    "tag_overrides": {}
+  },
+  "base_images": {
+    "base": {
+      "cluster": "https://api.ci.openshift.org",
+      "namespace": "openshift",
+      "name": "origin-v3.11",
+      "tag": "base"
+    }
+  },
+  "test_base_image": {
+    "cluster": "https://api.ci.openshift.org",
+    "namespace": "openshift",
+    "name": "release",
+    "tag": "golang-1.10"
+  },
+
+  "canonical_go_repository": "github.com/openshift/service-serving-cert-signer",
+
+  "images": [
+    {
+      "from": "base",
+      "to": "service-serving-cert-signer",
+      "inputs": {
+        "root": {"as": ["0"]}
+      }
+    }
+  ],
+
+  "tests": [
```

Comment:
will I get a test that tries to build the image too or is that only done when master moves?

Reply:
the unit test will also build the image, if either fails it'll fail